### DISCUSSION TITLE
Master duel had an engine update, unity engine was updated, and now I have the following issue on an M1 Pro

### DISCUSSION THREAD

**Discussion by josepheiba** [Whisky-App/Whisky#979] [Q&A]
I have installed it via Steam for windows. The game was running fine before the update, but right now, when the FPS drops bellow 120 FPS, everything appear in slow motion. I'm using the default settings.

**Comment by Lhaplus-K** [comment]
I am also experiencing this problem on some screens. In my case, slow motion occurs at the coin toss before the start of the duel and at the screen during the duel.
After some trials, I changed the window size to 2560 * 1440 and the symptoms went away. However, 3200 * 1800 is the maximum size in my environment and I would like to play with this window size if possible.
